client/src/App.tsx

```diff
@@ -22,6 +23,8 @@ function App(): JSX.Element {
               <Switch>
                 <Route exact path="/login" component={Login} />
                 <Route exact path="/signup" component={Signup} />
+                <Route exact path="/profile-settings" component={ProfileSettings} />
+                <Route exact path="/profile-settings/edit-profile" component={ProfileSettings} />
```

Comment:
Alternatively, we can use path parameters to make our life easier here :)

Eg: `<Route exact path="/profile-settings/:setting" component={ProfileSettings} />`